
This happened because of "downgrading" 8.3-STABLE to 9.0-RELEASE,
some commits which hadn't made it to 9.0-RELEASE were MFC'ed to
8.3-STABLE, adding new symbols. Upgrading to 9.0 removed them.

--- a/lib/libusbhid/Makefile
+++ b/lib/libusbhid/Makefile
@@ -5,7 +5,7 @@ LIB=	usbhid
 MAN=	usbhid.3
 
 LDADD= -lbsd
-SHLIB_MAJOR= 4
+SHLIB_MAJOR= 4debian
 
 MLINKS=	usbhid.3 libusbhid.3 usbhid.3 hid_get_report_desc.3 \
 	usbhid.3 hid_dispose_report_desc.3 \
